tests/test_sra_download_with_conda/test.local.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Tear down test environment
+cleanup () {
+    rc=$?
+    rm -rf .cache/
+    rm -rf .config/
+    rm -rf .fontconfig/
+    rm -rf .java/
+    rm -rf .snakemake/
+    rm -rf logs/
+    rm -rf results/
+    cd $user_dir
+    echo "Exit status: $rc"
+}
+trap cleanup EXIT
+
+# Set up test environment
+set -eo pipefail  # ensures that script exits at first command that exits with non-zero status
+set -u  # ensures that script exits when unset variables are used
+set -x  # facilitates debugging by printing out executed commands
+user_dir=$PWD
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+cd $script_dir
+
+# Run tests
+snakemake --snakefile="../../workflow/rules/sra_download.smk" \
+          --profile="../../profiles/local-conda" \
+          --config samples="../input_files/sra_samples.tsv" \
+                   outdir="results/sra_downloads" \
+                   samples_out="results/sra_downloads/sra_samples.out.tsv" \
+                   cluster_log_dir="logs/cluster_log"
```

Comment:
I guess the workflow is small enough so that we agreed _not to_ create configfiles for it, right?

Reply:
Yeah I think we don't need a configfile, that is why it is supplied necessary parameters directly in this call.